### DISCUSSION TITLE
binaries

### DISCUSSION THREAD

**Discussion by d7coders** [AcademySoftwareFoundation/openvdb#1409] [General]
hi,
why there is no pre build binaries?

**Comment by gondsm** [comment]
As far as I know, OpenVDB is pre-packaged for various Ubuntu distributions, including Xenial (but not Bionic, apparently) and available for installation through `apt-get`. Not sure about other distros/OSs, though.

EDIT: I was wrong, it is also packaged for [other Ubuntu releases](https://launchpad.net/ubuntu/+source/openvdb).

**Comment by dokipen3d** [comment]
There is also a 5.0 release on vcpkg which now cross platform. There is also brew on macos. I wonder if the steering  committee would be interested in maintaining the various package manager distributions.

**Comment by kmuseth** [comment]
Binary distribution is one of the topics I want to bring up in our next steering committee meeting (in a few hours:)

**Comment by dokipen3d** [comment]
Hi all :) are there any minutes from the meetings that could be shared publicly?

**Comment by kmuseth** [comment]
Notes are not up yet but here's the site: https://github.com/AcademySoftwareFoundation/tac/blob/master/meetings/2018-11-07.md
